swagger.json

```diff
@@ -3106,6 +3282,42 @@
                 }
             }
         },
+        "UnassignedDrivingSegment": {
+            "type": "object",
+            "description": "A driving segment that has no associated driver.",
+            "properties": {
```

Comment:
Could we give this unassigned_driving_segment resource a unique identifier, so it can be accessed via `/fleet/unassigned_driving_segments/:id`?